internal/testing/e2e/fetch_matches_test.go

```diff
@@ -529,7 +529,7 @@ func TestRegistrationWindow(t *testing.T) {
 		_, ok := <-in
 		require.False(t, ok)
 
-		require.True(t, time.Since(startTime) > time.Millisecond*100, "%s", time.Since(startTime))
```

Comment:
can you set them as constants instead of modifying them one by one?

Reply:
done